### PR TITLE
More redirect changes and small tweaks

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
More redirects and small tweaks for spelling and removing 'indices'

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/cluster-api/cluster-allocation.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ title: Cluster allocation explain
 nav_order: 10
 parent: Cluster APIs
 has_children: false
+redirect_from:
+ - - opensearch/rest-api/cluster-allocation/
```
Comment (Naarcha-AWS) [inline _api-reference/cluster-api/cluster-allocation.md:8]:
```suggestion
 - /opensearch/rest-api/cluster-allocation/
```

File: _api-reference/cluster-api/cluster-awareness.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ parent: Cluster APIs
 has_children: false
 redirect_from:
   - /api-reference/cluster-awareness/
+  - opensearch/rest-api/cluster-awareness/
```
Comment (Naarcha-AWS) [inline _api-reference/cluster-api/cluster-awareness.md:9]:
```suggestion
  - /opensearch/rest-api/cluster-awareness/
```

File: _api-reference/cluster-api/cluster-health.md [hunk 3]
```diff
@@ -4,7 +4,8 @@ title: Cluster health
 nav_order: 40
 parent: Cluster APIs
 has_children: false
-redirect_from: /api-reference/cluster-health/
+redirect_from: 
+ - opensearch/rest-api/cluster-health/
```
Comment (Naarcha-AWS) [inline _api-reference/cluster-api/cluster-health.md:8]: We shouldn't remove the old redirect. 

```suggestion
 - /api-reference/cluster-health/
 - /opensearch/rest-api/cluster-health/
```

**Review by Naarcha-AWS — APPROVED** [review]
Approved, but need to tweak the formatting on a couple of redirects before merging.

**Comment by hdhalter** [comment]
Thank you @Naarcha-AWS for catching those!

File: _api-reference/cluster-api/cluster-decommission.md [hunk 4]
```diff
@@ -6,6 +6,7 @@ parent: Cluster APIs
 has_children: false
 redirect_from: 
   - /api-reference/cluster-decommission/
+  - opensearch/rest-api/cluster-decommission/
```
Comment (kolchfa-aws) [inline _api-reference/cluster-api/cluster-decommission.md:9]:
```suggestion
  - /opensearch/rest-api/cluster-decommission/
```

File: _im-plugin/ism/managedindexes.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
 ---
 layout: default
-title: Managed Indices
+title: Managed Indexes
```
Comment (Naarcha-AWS) [inline _im-plugin/ism/managedindexes.md:3]:
```suggestion
title: Managed indexes
```

**Review by Naarcha-AWS — APPROVED** [review]

**Pull request merged**